### PR TITLE
Add By Mouth to prescription list options (Fixes #5714)

### LINKED ISSUE #5714: Add "By Mouth" to prescription details
### Describe the problem

*(As a quick note to start, this work is being driven by my employer, [Altispeed Technologies](https://altispeed.com/). I'm actively working on this issue. Any screenshots shown use test data, and are not associated with any real patient.)*

When entering a prescription, when the medication is taken orally, you can only specify the old-style wording, "Per Oris."

image

### Describe the solution you'd like

I'd like to have a By Mouth option available up by the Per Oris option.

### PR DESCRIPTION
**Pull request by tsimonq2**
Fixes #5714

### CODE REVIEW

File: sql/database.sql [hunk 1]
```diff
@@ -4073,6 +4073,7 @@ INSERT INTO list_options ( list_id, option_id, title, seq, is_default, notes ) V
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default, notes ) VALUES ('drug_route', 'transdermal', 'Transdermal', 19, 0, 'TD');
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default, notes, codes ) VALUES ('drug_route','intramuscular','Intramuscular' ,20, 0, 'IM', 'NCI-CONCEPT-ID:C28161');
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default, notes, codes ) VALUES ('drug_route','inhale','Inhale' ,16, 0, 'RESPIR', 'NCI-CONCEPT-ID:C38216');
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default, notes, codes ) VALUES ('drug_route', '21','By Mouth'         , 1,0, 'PO', 'NCI-CONCEPT-ID:C38288');
```
Comment (stephenwaite) [inline sql/database.sql:4076]: Looks good @tsimonq2 , just some minor formatting that would keep things consistent
```
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default, notes, codes ) VALUES ('drug_route', '21', 'By Mouth', 1, 0, 'PO', 'NCI-CONCEPT-ID:C38288');

File: sql/7_0_0-to-7_0_1_upgrade.sql [hunk 2]
```diff
@@ -192,4 +192,8 @@ ALTER TABLE `openemr_postcalendar_events` ADD `uuid` binary(16) DEFAULT NULL;
 
 #IfNotIndex openemr_postcalendar_events uuid
 CREATE UNIQUE INDEX `uuid` ON `openemr_postcalendar_events` (`uuid`);
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfNotRow list_options title "By Mouth"
```
Comment (stephenwaite) [inline sql/7_0_0-to-7_0_1_upgrade.sql:197]: probably safer to use
```
#IfNotRow2D list_options list_id drug_route option_id 21

Comment (bradymiller) [reply sql/7_0_0-to-7_0_1_upgrade.sql:197]: would use option_id bymouth

File: sql/database.sql (hunk 1 above)
Comment (bradymiller) [reply sql/database.sql:4076]: would use option_id bymouth

**Comment by tsimonq2** [comment]
How does that look?

**Comment by tsimonq2** [comment]
I realized I had a one-line bug, fixed. Let me know :)

**Comment by stephenwaite** [comment]
Looks good, thanks @tsimonq2!

**Pull request merged**